docs/features/tagging/high-level-design.md

```diff
@@ -0,0 +1,170 @@
+# Feature Tagging (High-Level Design)
+
+The selected solution for this feature is to implement a new table `feature_tags` on the API DB that will hold, for each
+row, a tag for a given combination of feature and project id like this:
+
+`| rowId (uuid) | projectId (uuid) | featureId (uuid) | tag (VARCHAR) | last_applied (timestamp) |`
```

Comment:
from the rest of the text, I suppose you meant

```suggestion
`| rowId (uuid) | projectId (uuid) | featureId (uuid) | tag (VARCHAR) | last_updated (timestamp) |`
```

It's just a tiny implementation detail, in any case (and maybe at implementation time I'd suggest to stick to the same name we have for similar columns already, that is `last_modified_at timestamp without timezone default now()`)

Reply:
Good catch, I'll change it later.